### ISSUE TITLE
Possible to use fusedio without an account?

### ISSUE THREAD

**Issue by paulterinho** [fusedio/udfs#317]
Hi Fusedio development team,

Is it possible to use the fusedio framework without create an account?  It looks like it is not possible, but wanted to make sure.

(Would love to contribute to this project, but I wanted to make sure it was open-source first. )

**Comment by pgzmnk** [comment]
Hi @paulterinho. We appreciate you checking in for clarification.

Anyone can indeed use Fused Python SDK without creating an account. Users without accounts can:

[Run UDFs](https://docs.fused.io/core-concepts/run/#ways-to-run-a-udf) using local resources. For example, to declare a UDF locally. 
```python
@fused.udf
def udf():
  import pandas as pd
  return pd.DataFrame({"a": [1, 2, 3]})
fused.run(udf, engine='local')
```

Run Public UDFs using Fused's realtime engine for free. For example, to call the "Overture Maps Example" UDF in this repo, passing a custom value to the `bbox` parameter.
```python
import fused
fused.run('UDF_Overture_Maps_Example', bbox=[-122.349, 37.781, -122.341, 37.818])
```

Any UDF you contribute to this repository will be runnable by anyone else using Fused cloud resources for free.

**Comment by paulterinho** [comment]
@pgzmnk Neat! So all the elements of the ecosystem are this way? We don't need to sign in to use things like the workbench?

**Comment by paulterinho** [comment]
@pgzmnk I might be missing something. I tried the code snippet you suggested, but in order to get the UDFs we'll need to use `load` method. This load method doesn't seem to have a parameter this prevents auto-login, and results in  'existing credentials not found`.

![Screenshot from 2024-10-07 17-15-57](https://github.com/user-attachments/assets/eb5c75af-f127-4643-8279-54abc2c4069b)

**Comment by pgzmnk** [comment]
@paulterinho Yes! To be clear, users don't need an account to run Public UDFs (those in this [repo](https://github.com/fusedio/udfs/tree/main/public)) using cloud resources. 

This means users can interact with public UDFs on Workbench without sign-in or invoke them from a Python environment. As an example, you could [add this UDF to workbench](https://www.fused.io/workbench/catalog/Overture_Maps_Example-64071fb8-2c96-4015-adb9-596c3bac6787).

This also means if someone [shares a UDF with the community](https://docs.fused.io/python-sdk/contribute/#share-a-udf-with-the-community), other folks can run it for free and invoke it with different parameters. 

On the other hand, folks looking to run UDFs with custom code can sign up for the waitlist.
